apps/src/maze/neighborhoodSprites.js

```diff
@@ -0,0 +1,1792 @@
+export const sheetRows = {
```

Comment:
Out of curiosity, why make this a js file instead of a json file?

Reply:
We need to send it as a js object to maze, so this removes the step of converting it from json to js in code

Reply:
If you want to keep it as json, I think you can import it to js by default in this way: https://github.com/code-dot-org/code-dot-org/blob/30655d5cee703d6d5e0ae7d3265b554bf656b0ac/apps/test/unit/StudioAppTest.js#L21

Reply:
oh cool, will update it to do that